actionlist/app/components/contentHeader.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+} from 'react-native';
+import { Colors } from '../lib/commons';
+
+const style = StyleSheet.create({
+  headerText: {
+    fontSize: 10,
+    color: Colors.IUCrimson,
+    fontFamily: 'BentonSansBold, Arial, sans-serif',
+    margin: 10,
+  },
+});
+
+const ContentHeader = ({ title }) => (
+  <View>
+    <Text style={style.headerText}>{title}</Text>
```

Comment:
I'd recommend not using a `title` prop and just using the `children` prop. It looks like it's allowed for `Text` components to have node children, so if you just use children you can pass off prop validation to the react-native component and allow yourself more flexibility.